docs/overview/trustbloc-did-method.md

```diff
@@ -78,12 +116,31 @@ Example of the format of the configuration data wrapped within the JWS:
 ###### Consortium Endorsement Signatures
 Stakeholders endorse a consortium configuration file using JWS multi-signature - they sign the JWS payload, with the consortium adding their signatures to the JWS.
 
-##### Stakeholder Config Files
+##### History
+The `history/` directory contains historical consortium configs. Each such file is named `[hash].json`, where `[hash]` is the SHA-256 hash of the given file.
```

Comment:
Should check with Bob on the DCAS REST API

Reply:
My intention was that the history retrieval would not have to shape its API around the DCAS, but allow implementers to use whatever storage method fits their needs. With the files being named and addressed using their hashes, and with the files being JWS-wrapped, I think the spec already asserts the integrity of history files, so an implementer doesn't even need to use a DCAS at all.